rust/connlib/clients/apple/src/lib.rs

```diff
@@ -229,6 +209,16 @@ impl WrappedSession {
         })
     }
 
+    fn reconnect(&mut self) {
+        tracing::debug!("Reconnecting");
+        self.inner.reconnect()
+    }
+
+    fn set_dns(&mut self, dns_servers: String) {
+        tracing::debug!("Setting DNS servers: {}", dns_servers);
+        // self.inner.set_dns(dns_servers)
```

Comment:
We shouldn't merge until this is wired up, otherwise it will break the apple client

Reply:
Can we merge a minimal version of `set_dns` that allows these PRs to be independent?

I think that would be useful.